src/app.py

```diff
@@ -5,35 +5,42 @@
 
 iexec_out = os.environ['IEXEC_OUT']
 iexec_in = os.environ['IEXEC_IN']
+dataset_filename = os.environ['IEXEC_DATASET_FILENAME']
+
+dataset_path = iexec_in + '/' + dataset_filename
+
+result = {"status_code": 0}
+dataset_exists = False
 
 # Eventually use some confidential assets
-if os.path.exists(iexec_in + '/data.json'):
-    with open(iexec_in + '/data.json', 'r') as json_file:
+if os.path.exists(dataset_path):
+    dataset_exists = True
+    with open(dataset_path, 'r') as json_file:
         secret = json.load(json_file)
```

Comment:
else write and quit asap?

Reply:
That would duplicate the `with open...` blocks. I prefer current version.

Reply:
Oh missed the indentation on
`mailjet = Client(auth=(secret["api_key"], secret["api_secret"]), version='v3.1')`
!